docs/operational_guide/bootstrapping.md

```diff
@@ -0,0 +1,104 @@
+# Bootstrapping
+
+## Introduction
+
+We recommend reading the [topology operational guide](topology.md) before reading the rest of this document.
+
+When an M3DB node is turned on (or experiences a topology change) it needs to go through a bootstrapping process to determine the integrity of data that it has. In most cases, as long as you're running with the default and recommended bootstrapper configuration of: "filesystem,commitlog,peers,uninitialized_topology" then you should not need to worry about the bootstrapping process at all and M3DB will take care of doing the rigth thing such that you don't lose data and its consistency guarantees are met.
```

Comment:
does ordering of `"filesystem,commitlog,peers,uninitialized_topology"` matter? If so that should be called out?